snapshot-labs/snapshot
# fix: Remove json field for plugins and more

Fixes #3459

## Changes 

- No longer shows json field on setup when `defaults.space` isn't defined inside the `plugin.json`
  - @zzuziak all the plugins that need the json field on setup already have `defaults.space` defined, but we should probably make it more clear in the docs that plugins that require additional json input should provide an example in `default.space`
  - The json field is still always accessible after selecting the plugin, inside the list used plugins
- Remove plugins from list of plugins when the plugin is used.
- Some other small bug fixes

## How to test
1. Open deployment on this PR
2. Go the bottom of space settings
3. See Plugins block
